packages/components/ng-ovh-sso-auth-modal-plugin/src/controller.js

```diff
@@ -41,13 +39,18 @@ export default /* @ngInject */ function SsoAuthModalController(
 
   function getUser() {
     const deferredObj = $q.defer();
-    $.ajax(self.data.userConfig)
-      .done((data) => {
-        deferredObj.resolve(data);
+
+    fetch(self.data.userConfig.url, {
```

Comment:
do you think fetch is supported by all browsers? or do we need a polyfill? IE does not support it https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API#browser_compatibility

Reply:
Hello @radireddy, seen with @cbourgois, the polyfill, if required should be handled by the app which use this module, not by the module itself. So it will use the fetch, regardless of the targeted browser

Reply:
Which means that the target should know it must handle fetch. I didn't check, is it specified anywhere?